apps/avatax/e2e/graphql/mutations/CreateCheckoutWithTwoItems.graphql

```diff
@@ -0,0 +1,28 @@
+mutation CreateCheckoutWithTwoLines(
+  $channelSlug: String!
+  $variantOneId: ID!
+  $variantTwoId: ID!
+  $email: String!
+  $address: AddressInput!
+) {
+  checkoutCreate(
+    input: {
+      channel: $channelSlug
+      lines: [
+        { quantity: 10, variantId: $variantOneId }
+        { quantity: 10, variantId: $variantTwoId }
```

Comment:
question: Why 10? Is there a reason we need to add additional quantity for that test? 🤔

Reply:
Not really, other tests used lines with quantity of 10 so I did the same